helm/templates/deployment.yaml

```diff
@@ -33,7 +33,11 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.deployment.securityContext | nindent 12 }}
+          {{- if .Values.image.ko }}
+          image: "ko://github.com/kro-run/kro/cmd/controller"
```

Comment:
interesting, TIL.

Reply:
oh if you are new to ko, buckle up. It is going to change your life.